TYPO3.Neos/Resources/Private/Templates/Media/Asset/RelatedDocumentNodes.html

```diff
@@ -0,0 +1,46 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="EditImage"/>
+
+<f:section name="Title">RelatedDocumentNodes view of AssetController</f:section>
+
+<f:section name="Content">
+      <legend>{asset.resource.filename}</legend>
+
+      <table class="neos-info-table">
+        <thead>
+          <tr>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.document', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th>{neos:backend.translate(id: 'media.relatedDocumentNodes.reference', source: 'Modules', package: 'TYPO3.Neos')}</th>
+            <th class="neos-aRight">{neos:backend.translate(id: 'media.field.lastModified', source: 'Modules', package: 'TYPO3.Neos')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <f:for each="{relatedDocumentNodes}" as="relatedDocumentNode">
+            <tr>
+              <td><neos:link.node node="{relatedDocumentNode.documentNode}" /></td>
+              <td>
+                <ul>
+                  <f:for each="{relatedDocumentNode.node}" as="node">
+                    <li>
+                      <f:if condition="{node.nodeType.label}">
+                        <f:then>
+                          <neos:backend.translate id="{node.nodeType.label}" /> ({node.nodeType.name})
+                        </f:then>
+                        <f:else>
+                          {node.nodeType.name}
+                        </f:else>
```

Comment:
would also be optimal if the node was linked bringing you to the page in your own workspace so you can remove the reference